templates/compute/data-sources/vkcs_compute_flavor.md.tmpl

```diff
@@ -11,6 +11,13 @@ description: |-
 {{ .Description }}
 
 ## Example Usage
+### Basic instance
 {{tffile "examples/compute/flavor/name/main.tf"}}
 
+### Instance with vcpus, ram and cpu_generation
+{{tffile "examples/compute/flavor/main.tf"}}
+
+### Instance with vcpus and min_ram
```

Comment:
It's not an instance, should be something like `Filter by number of vCPUs and minimum RAM`, same way in the other titles.